Dockerfile

```
@@ -0,0 +1,32 @@
+FROM alpine:3.12
```

Maybe it would be better to start from Python images instead of using Alpine and then installing Python? I'd use python image instead here:

```suggestion
FROM python:3.8.6-alpine
```